app/views/error_wrapper.txt

```diff
@@ -73,8 +73,8 @@
 
 
 @scriptElement = {
-<script src='@routes.Assets.at("jquery/jquery-ui-1.12.1.min.js")'></script>
-<script src='@routes.Assets.at("javascript/timeout-dialog.js")'></script>
+<script src='@routes.Assets.versioned("jquery/jquery-ui-1.12.1.min.js")'></script>
```

Comment:
So sort of same question - not sure why we are using different versions for different files ? Maybe a technical reason ?

Reply:
File deleted - so no longer relevant